### PR TITLE
Add style parameter to MarkupText

### LINKED ISSUE #381: [leancode_markup] Add style parameter to MarkupText
Currently, `MarkupText` doesn't provide a `style` parameter that would allow setting a default initial style for our text.

It's especially problematic if you have multiple custom fonts in the app and you want to use rich text with a custom font or additional effects for a single widget.

For example:
```dart
return MarkupText(
  'hello [b]world[/b]',
  // This parameter doesn't exist.
  style: const TextStyle(
    fontFamily: 'ComicSansMS',
    shadows: [
      // Some shadows...
    ],
  ),
);
```

### PR DESCRIPTION
**Pull request by n-bernat**
Resolves #381

image

### CODE REVIEW

**Review by pdenert — COMMENTED** [review]
It looks good. Please update the changelog

**Comment by n-bernat** [comment]
Who is responsible for tagging new versions after merging? Codeowner?

File: packages/leancode_markup/CHANGELOG.md
Comment (pdenert) [inline packages/leancode_markup/CHANGELOG.md:1]: It's a feature, not fix, so it should be 0.1.0

File: packages/leancode_markup/pubspec.yaml
Comment (pdenert) [inline packages/leancode_markup/pubspec.yaml:2]: ditto

**Comment by pdenert** [comment]
> Who is responsible for tagging new versions after merging? Codeowner?

If you're changes are approved, then you can release it

File: packages/leancode_markup/CHANGELOG.md
Comment (pdenert) [inline packages/leancode_markup/CHANGELOG.md:3]: One last thing - we should write changelog in imperative verb form, same like in commits
```suggestion
- Add `style` parameter to `MarkupText`.
```

**Review by pdenert — APPROVED** [review]
lgtm

**Pull request merged**
